src/Services/Passenger/src/Passenger/Passengers/Features/CompletingRegisterPassenger/V1/CompleteRegisterPassenger.cs

```diff
@@ -95,16 +96,19 @@ public CompleteRegisterPassengerCommandHandler(IMapper mapper, PassengerDbContex
     {
         Guard.Against.Null(request, nameof(request));
 
-        var passenger = await _passengerDbContext.Passengers.AsNoTracking().SingleOrDefaultAsync(
-            x => x.PassportNumber == request.PassportNumber, cancellationToken);
+        //var passenger = await _passengerDbContext.Passengers.AsNoTracking().SingleOrDefaultAsync(
```

Comment:
Why you have commented code? Why You change AsNoTracking() to AsEnumerable()?

Reply:
because i got error this:
No backing field could be found for property 'PassportNumber.PassengerId' and the property does not have a getter.

but when i get the last result(AsEnumerable()) and then i filtered it i got no error 
(instead of filtering meanwhile of getting result i think.)

I revert my changes as you commented above

Reply:
done